torch/_inductor/kernel/mm_plus_mm.py

```diff
@@ -163,12 +162,13 @@ def tuned_mm_plus_mm(mat1, mat2, mat3, mat4, *, layout=None):
     choices = [aten_mm_plus_mm.bind((mat1, mat2, mat3, mat4), layout)]
     if use_triton_template(layout):
         for config in mm_configs():
-            choices.append(
-                mm_plus_mm_template.generate(
-                    (mat1, mat2, mat3, mat4),
-                    layout,
-                    **mm_options(config, k, layout),
+            if config.kwargs['BLOCK_K'] < k:
```

Comment:
Maybe add a comment with a pointer to the triton issue so we can revisit someday.

Reply:
Added